### DISCUSSION TITLE
my monika know only one horror story

### DISCUSSION THREAD

**Discussion by LukkYT01** [Monika-After-Story/MonikaModDev#9653] [QA - tech support + general] [answered]
i ask every day a new horror story from monika but she doesn't know any other horror stories, only one: The Hunter. she tells me short stories but not horror ones. can someone help me?
![Immagine 2022-10-08 204242](https://user-images.githubusercontent.com/114704443/194722810-c2b75e4a-e98a-443e-9294-6b899c248ef4.png)
![screenshot0001](https://user-images.githubusercontent.com/114704443/194722814-b2db7983-30a6-478f-9769-2042eb5a2d14.png)

**Comment by Booplicate** [comment, marked as answer]
Have you ever time travelled?

**Reply by LukkYT01** [reply]
no. what is that?

**Reply by Booplicate** [reply]
Send your persistent (you may need to add in into a `zip`).

**Reply by Booplicate** [reply]
You've tried to cheat the system and changed your clock to visit her on her birthday. As you've been warned, time travelling may break the mod, which is probably what happened. This might or might not fix itself after some time, but some other things may still not work as intended.

**Comment by LukkYT01** [comment]
[Nuovo WinRAR archive.zip](https://github.com/Monika-After-Story/MonikaModDev/files/9740361/Nuovo.WinRAR.archive.zip)
